src/docs/Foundations/Typography/HeadingStyles/index.mdx

```diff
@@ -1,49 +1,92 @@
 ---
 title: 'Heading styles'
 order: 5
+description: 'Heading patterns provide basic typographic styles for your titles to help the user to quickly scan main informations of the page or section. It creates hierarchy in your pages or patterns'
 status:
   sketch: 'wip'
   scss: 'wip'
 ---
 
-> The heading patterns provide basic typographic styles for headings.
+> Heading patterns provide basic typographic styles for your titles to help the user to quickly scan main informations of the page or section. It creates hierarchy in your pages or patterns.
 
-## Usage
+## Recommended sizes
 
-They are destined to style mono-line (or few lines) headings and help the user to quickly scan major informations of the page or section, and to create hierarchy between them.
+### Heading Medium
 
-Heading styles are patterns meant to help you use consistent typographic hierarchy with the context and across responsive viewports. It's mainly reserved for users wishing to focus on tasks with added value and user experience. Text classes are families of several font-sizes and help create a hierarchy of contents.
+These are the default values for your heading styles for every breakpoint.
 
-## Headings styles define three properties of a text :
+| Breakpoint              | Heading size    |
+| ----------------------- | --------------  |
+| `S` (320px and up)      | 28px (1.75mu)   |
+| `M` - (680px and up)    | 34px (2.125mu)  |
+| `L` - (1024px and up)   | 41px (2.5625mu) |
+| `XL` - (1280px and up)  | 41px (2.5625mu) |
+| `XXL` - (1920px and up) | 41px (2.5625mu) |
```

Comment:
It may not be necessary to specify these breakpoints, since there is no change from `L`

Reply:
even if it's the same piece of information I think designers will think that this is missing if we don't write it down 😉

Reply:
"XXXpx and up" already mean what it mean